### DISCUSSION TITLE
How the "H" Do I remove this new "@#$%" from my home page?<title>

### DISCUSSION THREAD

**Discussion by chindraba-work** [community/community#41872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I use GH for managing code, not as a social network platform. How do I remove this tripe (dashboard, for you, feed, whatever the "F" you call it, ALL of it) from the page and just have a list of my active repos, or something useful as a developer, not a social butterfly?

**Comment by radar-blip** [comment]
Just log into your profile page and skip all that stuff. Maybe keep your profile book marked.

**Comment by Vonjay77** [comment]
Javaka
